src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        excavators.add(new Excavator()); // Додаємо новий Excavator до списку
```

Comment:
The `get` method currently adds only one `Excavator` to the list. If the task requires producing multiple excavators, consider adding more instances to the list.